lib/src/play_store_search_api.dart

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021 William Kwabla. All rights reserved.
+ */
+
+import 'package:html/dom.dart';
+import 'package:html/parser.dart' show parse;
+import 'package:http/http.dart' as http;
+
+class PlayStoreSearchAPI {
+  /// Play Store Search Api URL
+  final String playStorePrefixURL = 'play.google.com';
+
+  /// Provide an HTTP Client that can be replaced for mock testing.
+  http.Client? client = http.Client();
+
+  bool debugEnabled = false;
+
+  /// Look up by id.
+  Future<Document?> lookupById(String id) async {
+    if (id.isEmpty) {
+      return null;
+    }
+
+    final url = lookupURLById(id)!;
+
+    final response = await http.get(url);
```

Comment:
@Williano Instead of using `http` here, it should use `client`, so it can be mocked.

Reply:
When I use the client!.get(url), it returns null but when I use the http.get(url) it returns the HTML content.